.github/workflows/soak_infra.yml

```diff
@@ -107,13 +109,13 @@ jobs:
             org.opencontainers.image.description=Base image for use with building soak-vector images
             org.opencontainers.image.title=Soak Builder
 
-      - name: Build and push Docker image
+      - name: Build and push Docker image (linux/arm64)
         uses: docker/build-push-action@v2.10.0
         with:
           context: .
           file: soaks/Dockerfile.builder
-          push: ${{ github.ref == 'refs/heads/master' }}
-          platforms: linux/amd64,linux/arm64
+          push: true
```

Comment:
Did you mean to update the `push` here to always push?

Reply:
Ah, I'll put this back before merging. I did need to always push while developing.